src/components/calcite-input/calcite-input.tsx

```diff
@@ -355,7 +358,7 @@ export class CalciteInput {
   //--------------------------------------------------------------------------
 
   private clearInputValue = (nativeEvent: KeyboardEvent | MouseEvent): void => {
-    this.setValue("", nativeEvent, true);
+    this.setValue({ value: "", nativeEvent, committing: true });
```

Comment:
For internal methods, we can keep arguments vs a parameter object, as long as it doesn't get unwieldy (around 3, 4 is the limit for me). Reasoning: it avoids creating an object for frequently invoked methods.